packages/flutter/lib/src/rendering/proxy_box.dart

```diff
@@ -2352,16 +2352,30 @@ class RenderFittedBox extends RenderProxyBox {
     markNeedsPaint();
   }
 
+  bool _fitAffectsLayout(BoxFit fit) {
+    switch(fit) {
```

Comment:
trivial nit: please be consistent about whether a space appears after `switch` (space preferred)